deepfence_frontend/apps/dashboard/src/types/common.ts

```diff
@@ -15,3 +17,23 @@ export type PostureSeverityType =
   | 'pass'
   | 'warn'
   | 'note';
+
+export enum VulnerabilityScanNodeTypeEnum {
+  host = 'host',
+  kubernetes_cluster = 'kubernetes_cluster',
+  registry = 'registry',
+  image = 'image',
+  imageTag = 'imageTag',
+}
+
+export { VulnerabilityScanNodeTypeEnum as MalwareScanNodeTypeEnum };
```

Comment:
I think these two exports is not used MalwareScanNodeTypeEnum and SecretScanNodeTypeEnum

Reply:
they are used